### DISCUSSION TITLE
Plugin.LocalNotification Queue + Notifications Blocked on Phone

### DISCUSSION THREAD

**Discussion by ckoziel** [thudugala/Plugin.LocalNotification#482] [Q&A]
Thanks for responding to my email Elvin. So, my question was this. If you schedule notifications when the Phone Notifications are Blocked, I was wondering if the notifications still get queued for when the Notifications are eventually allowed. Here are the steps that I tried. 

1) Ran the APP in debug mode on a physical Android Device
2) Scheduled the notifications with the Notification Blocked on the Phone
3) The result comes back false.
4) Turn the Notifications on the phone and nothing gets fired.

1) If the Notifications are allowed on the phone, and then I schedule the notifications, everything runs fine.
Maybe it's just my misunderstanding, but I was under the impression that notifications still get queued even if Notifications are blocked.
[TestNotifications.zip](https://github.com/thudugala/Plugin.LocalNotification/files/14381330/TestNotifications.zip)

Please see the attached sample. Thank you, Elvin.

**Comment by ckoziel** [comment]
Nuget Version 9.2.0 will schedule a notification when the device notifications are Blocked.
 var CheckStatus = await NotificationCenter.Current.Show(notification); :: returns true

Nuget Version 10.1.8 will not schedule a notification when the device notifications are Blocked.
 var CheckStatus = await NotificationCenter.Current.Show(notification); :: returns false

**Comment by thudugala** [comment]
@ckoziel Yes, from 10.1.8 onwards, if notifications are not allowed, you cannot schedule notifications

![image](https://github.com/thudugala/Plugin.LocalNotification/assets/4112014/490ed646-91d1-411d-a93d-4499e9f412d4)
